lib/main.js

```diff
@@ -5,10 +5,11 @@ var constants = require('./constants')
 
 const _PACKAGE_VERSION = require('../package.json').version
 
-function Client (ikey, skey, host) {
+function Client (ikey, skey, host, sigVersion = 2) {
```

Comment:
not critical, but should we replace the magic number with a const like
SIGNATURE_VERSION_2 / SIGNATURE_VERSION_5 to make it clearer what this represents?

Reply:
Now exporting `SIGNATURE_VERSION_2` and `SIGNATURE_VERSION_5`, as well as using them internally.